app/controllers/subdomains_controller.rb

```diff
@@ -2,7 +2,8 @@ class SubdomainsController < ApplicationController
   include Subdomainable
 
   skip_before_action :verify_authenticity_token
-  before_filter :set_app, :app_authorize!
+  prepend_before_filter  :set_app
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.